asgs.ttl

```diff
@@ -1957,11 +1983,16 @@ asgs:StatisticalAreaLevel1 rdf:type owl:Class ;
                                            [ rdf:type owl:Restriction ;
                                              owl:onProperty asgs:isStatisticalAreaLevel1Of ;
                                              owl:allValuesFrom asgs:MeshBlock
+                                           ] ,
+                                           [ rdf:type owl:Restriction ;
```

Comment:
i assume this will be the pattern going forward to align this with other things that use StatisticalAreaLevel1, as described in https://www.abs.gov.au/websitedbs/D3310114.nsf/home/Australian+Statistical+Geography+Standard+(ASGS)

Reply:
... uses `asgs:within` uniformly to constrain the containment relationships between types, per the Fig 1. diagram and documentation

Reply:
The next step might be to remove all the `asgs:isXXXXXof` which are sub-properties of `asgs:contains` but much more noisy than these `asgs:within` structures.